xml/System.Net/HttpStatusCode.xml

```diff
@@ -463,7 +464,7 @@ If the <xref:System.Net.HttpWebRequest.AllowAutoRedirect%2A?displayProperty=name
       </ReturnValue>
       <MemberValue>424</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Equivalent to HTTP status 424. <see cref="F:System.Net.HttpStatusCode.FailedDependency" /> indicates that the method could not be performed on the resource because the requested action depended on another action and that action failed.</summary>
```

Comment:
```suggestion
        <summary>Equivalent to HTTP status code 424. Indicates that the method couldn't be performed on the resource because the requested action depended on another action and that action failed.</summary>
```